### PR TITLE
Put braces in one-line blocks issue #18

### PR DESCRIPTION
**Pull request by paulohrpinheiro**
Hi!

This pr is a initial work in issue #18 , to put braces in all one-line blocks. This first commits is only in src directory. Remains to be done the lib directory.

If this is OK, then I work in the lib directory.

Thanks!

### CODE REVIEW

File: src/tlog-test-json-stream.c [hunk 1]
```diff
@@ -232,52 +233,65 @@ test(const char *n, const struct test t)
...
                 FAIL_OP("rem_off %zd != %zd",
                         (last_rem - meta.rem), op->data.write.rem_off);
+            }
             last_rem = meta.rem;
-            if (passed)
+            if (passed) {
                 break;
-            else
+            }
+            else {
```
Comment (spbnick) [inline src/tlog-test-json-stream.c]: Could you please put else and braces on a single line, like this:

    } else {

**Comment by spbnick** [comment]
Oh, and yes, if you could do this for all the code, that would be great :)!

**Comment by paulohrpinheiro** [comment]
Hi! Yes, I do this.

--
Paulo Henrique Rodrigues Pinheiro
https://about.me/paulohrpinheiro

Em 07-10-2016 03:07, Nikolai Kondrashov escreveu:

**Comment by paulohrpinheiro** [comment]
Hi!

I appreciate the opportunity to collaborate on this project. I was 
wondering whether I was doing the else that way or not, thanks for 
clarifying. And thanks for the help, because I am still new in this 
world of collaboration :) And congratulations on your code, it is very 
well written and organized.

I will make a new commit according to these indications, and this time, 
with all the code, in one commit.

Thanks!
--
Paulo Henrique Rodrigues Pinheiro
https://about.me/paulohrpinheiro

Em 07-10-2016 03:11, Nikolai Kondrashov escreveu:

**Comment by paulohrpinheiro** [comment]
OK. Thanks!

Paulo Henrique Rodrigues Pinheiro
https://about.me/paulohrpinheiro

Em 07-10-2016 03:13, Nikolai Kondrashov escreveu:

**Comment by spbnick** [comment]
Thanks for the compliment. Sure, feel free to ask any questions regarding the code and contribution in general! I will be waiting for the updated changes!

Regarding pull requests in general, you don't have to close them, if you want to make some changes. Just rewrite your history locally however you wish, and then force-push it to the same branch you created the pull request from. The pull request will then be updated.

However, it's not a problem to start a new pull request for this, either.

**Comment by paulohrpinheiro** [comment]
Hi!


Sorry, I blacked out my branch and put in a new one, I did not know he 
could do that way.

But I have a question: in the case below, I place an empty block, or 
leave as is (lib/errs.c)?


         for (last_errs = errs;
              last_errs->next != NULL;
              last_errs = last_errs->next);


Thanks!

--
Paulo Henrique Rodrigues Pinheiro
https://about.me/paulohrpinheiro

Em 07-10-2016 07:48, Nikolai Kondrashov escreveu:

**Comment by spbnick** [comment]
It's OK about the branch and the pull-request, not a problem.

Regarding the loop, I'll ask around at work what people think is best, and especially the people who suggested this convention. Stay tuned!

**Comment by spbnick** [comment]
Alright, I would say let's leave it as is, without the body for now. Perhaps an empty body is more confusing than no body in this case. Thanks!

**Comment by paulohrpinheiro** [comment]
OK!

Paulo Henrique Rodrigues Pinheiro
https://about.me/paulohrpinheiro

Em 07-10-2016 10:22, Nikolai Kondrashov escreveu:

**Pull request closed without merging**